query/src/test/java/org/infinispan/query/distributed/MassIndexingTest.java

```diff
@@ -96,7 +96,7 @@ public boolean isSuccess(CompletionStage<Void> future) {
       try {
          FunctionalTestUtils.await(future);
          return true;
-      } catch (Exception e) {
+      } catch (Error e) {
```

Comment:
Why not ```catch Throwable``` ? I have strange gut feelings for Error

Reply:
From the ```Error``` javadoc:  "An Error is a subclass of Throwable that indicates serious problems that a reasonable application should not try to catch."  But tests are usually unreasonable applications because they are supposed to break and burn. :)

Reply:
```Throwable``` works too. Changed.